config/manager/manager.yaml

```diff
@@ -54,6 +54,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: POD_NAME
```

Comment:
@djzager what needs to be done to get this into the bundle? We'll also need to make sure it gets into ACM's chart, which might just require re-generating it from our updated bundle. But we should give them a heads-up about the change.

Reply:
Need to run `make generate-manifests` (that's why pr-merge is failing) to get it into the clusterserviceversion. Next time we push to community-operators-prod, ACM will be able to pick it up.